.rosinstall.master

```diff
@@ -14,7 +14,3 @@
     uri: https://github.com/aws-robotics/monitoringmessages-ros2.git
     local-name: monitoringmessages-ros2
     version: master
-- git:
-    uri: https://github.com/ros-tooling/system_metrics_collector.git
-    local-name: system_metrics_collector-ros2
-    version: master
```

Comment:
Let's just change `master` to `0863593226f65d2a9cbbb105e585f59d008d3861`, and discard the other changes.

Reply:
I'm up for that, my concern is that if there are changes to the `metrics_statistics_msgs` after that commit then we won't be able to pull in those changes because the rest of the `systems_metrics_collector` won't compile

Reply:
`metrics_statistics_msgs` and `systems_metrics_collector` are in the same repo. Otherwise, I wouldn't have even pulled in `systems_metrics_collector` into the `cloudwatchmetrics-ros2` workspace.

Reply:
I understand, what I'm trying to say is that because they are in the same repo we have to pull them both in and build both of them and so this package needs to be compatible with both of them. That happens to be true at commit `0863593226f65d2a9cbbb105e585f59d008d3861`, but if there are any changes to metric_statistics_msgs then we won't be able to pull those in leaving master in a state where it doesn't work with the master of `systems_metrics_collector`